spec/jobs/sms_account_reset_cancel_job_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe SmsAccountResetCancelJob do
+  include Features::ActiveJobHelper
+  include Rails.application.routes.url_helpers
```

Comment:
I don't think these are needed.

Reply:
reset_job_queues needed ActiveJobHelper.  I'll remove the url_helpers